dist/scripts/datalayer/common.js

```diff
@@ -2,27 +2,27 @@
 
 window.dataLayer = window.dataLayer || [], document.shopAnalytics = {
     getProductsData: function(t) {
-        var a = [];
+        var e = [];
```

Comment:
Unrelated to this PR: Wrong uglify settings: As long as the compiled file is committed, variable names must not be shortened/optimized.

Reply:
If I am afraid this is working like that in all our projects 🙈

Reply:
That is not true.  For example, in https://github.com/netzstrategen/wordpress-core-standards/blob/master/gulpfile.js#L38-L40 we are only enabling beautification (i.e., removal of comments and reformatting of code) but no further optimizations and minifications like variable name minifications.  It's the minification that rewrites the whole script file every time.

Reply:
Thanks for pointing that to me, @sun